pkg/common-controller/snapshot_controller.go

```diff
@@ -1281,6 +1281,9 @@ func (ctrl *csiSnapshotCommonController) getSnapshotFromStore(snapshotName strin
 func (ctrl *csiSnapshotCommonController) setAnnVolumeSnapshotBeingDeleted(content *crdv1.VolumeSnapshotContent) error {
 	// Set AnnVolumeSnapshotBeingDeleted if it is not set yet
 	if !metav1.HasAnnotation(content.ObjectMeta, utils.AnnVolumeSnapshotBeingDeleted) {
+		if content == nil {
+			return nil
+		}
```

Comment:
This check should be moved before the if block.  Otherwise we'll still get a nil exception on line 1283 as it references the content.  I'm going to revert this PR.  Can you please submit a different PR?

Reply:
Fixed in https://github.com/kubernetes-csi/external-snapshotter/pull/279